### PR TITLE
docs: address getting started feedback

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR addresses the feedback received from users who went through the new **Getting Started** section.
One example is the feedback left by @benradstone, which can be found [here](https://docs.google.com/document/d/1KD0MPTFePb08mYHfK4v1OYmRf_2saZIqqCh1WMOO5Fc/edit?tab=t.0#heading=h.lez85r7tlwi6).

## Changed Pages

💻 [Getting Started](https://deploy-preview-4809--docs-spectrocloud.netlify.app/getting-started/)

## Jira Tickets

🎫 [DOC-1493](https://spectrocloud.atlassian.net/browse/DOC-1493)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1493]: https://spectrocloud.atlassian.net/browse/DOC-1493?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by benradstone — APPROVED** [review]
Great changes!

File: docs/docs-content/getting-started/aws/aws.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ introducing you with Palette workflows and capabilities.
 In this section, you learn how to create a cluster profile. Then, you deploy a cluster to AWS by using Palette. Once
 your cluster is deployed, you can update it using cluster profile updates.
 
+<!-- vale off -->
```
Comment (addetz) [inline docs/docs-content/getting-started/aws/aws.md:25]: This never occurred to me LMAO

File: docs/docs-content/getting-started/aws/create-cluster-profile.md [hunk 2]
```diff
@@ -44,24 +44,25 @@ In the **Basic Information** section, assign the name **aws-profile**, a brief p
...
-| csi-aws-ebs    | 1.26.x  | Storage          |
+| Pack Name                   | Version | Registry    | Layer            |
+| --------------------------- | ------- | ----------- | ---------------- |
+| Ubuntu                      | 22.4.x  | Public Repo | Operating System |
+| Palette eXtended Kubernetes | 1.29.x  | Public Repo | Kubernetes       |
+| Calico                      | 3.27.x  | Public Repo | Network          |
+| Amazon EBS CSI              | 1.26.x  | Public Repo | Storage          |
 
-As you fill out the information for each layer, click on **Next** to proceed to the next layer.
+As you fill out the information for each layer, click on **Next Layer** to proceed to the next layer.
```
Comment (addetz) [inline docs/docs-content/getting-started/aws/create-cluster-profile.md:63]: Now that I think about it, what information is there to fill out here?

File: docs/docs-content/getting-started/aws/create-cluster-profile.md [hunk 3]
```diff
@@ -101,7 +102,7 @@ Select the **Enable Hello Universe API** preset. The pack manifest changes accor
 The pack requires two values to be replaced for the authorization token and for the database password when using this
 preset. Replace these values with your own base64 encoded values. The
 [_hello-universe_](https://github.com/spectrocloud/hello-universe?tab=readme-ov-file#single-load-balancer) repository
-provides a token that you can use.
+provides an unencoded that you can use.
```
Comment (addetz) [inline docs/docs-content/getting-started/aws/create-cluster-profile.md:105]:
```suggestion
provides an unencoded token that you can use.
```

File: docs/docs-content/getting-started/azure/create-cluster-profile.md [hunk 4]
```diff
@@ -43,24 +43,25 @@ In the **Basic Information** section, assign the name **azure-profile**, a brief
...
+| Ubuntu                      | 22.4.x  | Public Repo | Operating System |
+| Palette eXtended Kubernetes | 1.30.x  | Public Repo | Kubernetes       |
+| Calico                      | 3.26.x  | Public Repo | Network          |
+| Azure Disk                  | 1.28.x  | Public Repo | Storage          |
 
-As you fill out the information for each layer, click on **Next** to proceed to the next layer.
+As you fill out the information for each layer, click on **Next Layer** to proceed to the next layer.
 
-Click on **Confirm** after you have completed filling out all the core layers.
+Click on **Confirm** and then **Next** after you have completed filling out all the core layers.
```
Comment (addetz) [inline docs/docs-content/getting-started/azure/create-cluster-profile.md:64]:
```suggestion
Click on **Confirm** and then **Next** after you have completed selecting all the core layers.
```

File: docs/docs-content/getting-started/gcp/create-cluster-profile.md [hunk 5]
```diff
@@ -43,25 +43,26 @@ In the **Basic Information** section, assign the name **gcp-profile**, provide a
...
+| Ubuntu                      | 22.4.x  | Public Repo | Operating System |
+| Palette eXtended Kubernetes | 1.28.x  | Public Repo | Kubernetes       |
+| Calico                      | 3.27.x  | Public Repo | Network          |
+| GCE Persistent Disk CSI     | 1.12.x  | Public Repo | Storage          |
 
-As you fill out the information for each layer, click on **Next** to proceed to the next layer.
+As you fill out the information for each layer, click on **Next Layer** to proceed to the next layer.
 
-Click on **Confirm** after you have completed filling out all the core layers.
+Click on **Confirm** and then **Next** after you have completed filling out all the core layers.
```
Comment (addetz) [inline docs/docs-content/getting-started/gcp/create-cluster-profile.md:65]:
```suggestion
Click on **Confirm** and then **Next** after you have completed selecting all the core layers.
```

File: docs/docs-content/getting-started/vmware/create-cluster-profile.md [hunk 6]
```diff
@@ -52,18 +52,19 @@ The **Profile Layers** step is where you specify the packs that compose the prof
...
+| Ubuntu                      | 22.4.x  | Public Repo | Operating System |
+| Palette eXtended Kubernetes | 1.28.x  | Public Repo | Kubernetes       |
+| Calico                      | 3.27.x  | Public Repo | Network          |
+| vSphere CSI                 | 3.1.x   | Public Repo | Storage          |
 
-As you fill out the information for each layer, click on **Next** to proceed to the next layer.
+As you fill out the information for each layer, click on **Next Layer** to proceed to the next layer.
 
-Click on **Confirm** after you have completed filling out all the core layers.
+Click on **Confirm** and then **Next** after you have completed filling out all the core layers.
```
Comment (addetz) [inline docs/docs-content/getting-started/vmware/create-cluster-profile.md:67]:
```suggestion
Click on **Confirm** and then **Next** after you have completed selecting all the core layers.
```

**Review by addetz — APPROVED** [review]
Awesome changes! 🥳

File: docs/docs-content/getting-started/aws/aws.md (hunk 1 above)
Comment (caroldelwing) [reply docs/docs-content/getting-started/aws/aws.md:25]: :( I think that rule was introduced recently

**Pull request merged**